src/status_im/chat/events/input.cljs

```diff
@@ -403,20 +405,21 @@
 
 (handlers/register-handler-fx
   ::send-command
-  [re-frame/trim-v]
-  (fn [{{:keys [current-public-key current-chat-id]
-         :accounts/keys [current-account-id] :as db} :db} [{:keys [command] :as command-message}]]
-    {:db (-> db
-             clear-seq-arguments
-             (set-chat-input-metadata nil)
-             (set-chat-input-text nil)
-             (model/set-chat-ui-props {:sending-in-progress? false}))
-     ;; TODO: refactor send-message.cljs to use atomic pure handlers and get rid of this dispatch
-     :dispatch [:check-commands-handlers! {:message (get-in db [:chats current-chat-id :input-text])
-                                           :command  command-message
-                                           :chat-id  current-chat-id
-                                           :identity current-public-key
-                                           :address  current-account-id}]}))
+  message-model/send-interceptors
+  (fn [cofx [{:keys [command] :as command-message}]]
+    (let [{{:keys [current-public-key current-chat-id]
+            :accounts/keys [current-account-id] :as db} :db} cofx
+          cofx' (message-model/process-command cofx
+                                               {:message (get-in db [:chats current-chat-id :input-text])
+                                                :command command-message
+                                                :chat-id current-chat-id
+                                                :identity current-public-key
+                                                :address current-account-id})]
+      (assoc cofx' :db (-> (:db cofx')
```

Comment:
there is no reason to return `cofx` it's confusing, better to return `effects` or `fx`